src/stan/mcmc/hmc/nuts/base_nuts.hpp

```diff
@@ -152,14 +152,16 @@ namespace stan {
         names.push_back("stepsize__");
         names.push_back("treedepth__");
         names.push_back("n_leapfrog__");
-        names.push_back("n_divergent__");
+        names.push_back("divergent__");
+        names.push_back("E__");
```

Comment:
I don't think it will be a big deal.

On Thu, Mar 10, 2016 at 10:27 AM, Daniel Lee <notifications@github.com>
wrote:

> In src/stan/mcmc/hmc/nuts/base_nuts.hpp
> <https://github.com/stan-dev/stan/pull/1791#discussion_r55695430>:
>
> > @@ -152,14 +152,16 @@ namespace stan {
> >          names.push_back("stepsize__");
> >          names.push_back("treedepth__");
> >          names.push_back("n_leapfrog__");
> > -        names.push_back("n_divergent__");
> > +        names.push_back("divergent__");
> > +        names.push_back("E__");
>
> Summary:
>
>    - n_divergent__ moves to divergent__
>    - E__ is inserted
>
> I just need to know what sort of impact this has. It's easy for CmdStan.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stan-dev/stan/pull/1791/files#r55695430>.
>